spirv_hlsl.cpp

```diff
@@ -2734,7 +2734,7 @@ void CompilerHLSL::emit_buffer_block(const SPIRVariable &var)
 			for (auto &member : type.member_types)
 			{
 				add_member_name(type, i);
-				auto backup_name = get_member_name(type.self, i);
+				const auto &backup_name = get_member_name(type.self, i);
```

Comment:
This is still broken.

Reply:
fix it https://github.com/KhronosGroup/SPIRV-Cross/pull/2245/commits/707f8c6e8395cb319c1abfd301d1bbed02637998